doc: Coding_convention example changed

eduazocar opened the pull request:
<!--
We cares a lot about code quality.
Therefore, before describing what your contribution is about, we would like
you to make sure that your modifications are compliant with our coding conventions
-->

### Contribution description
 In this PR will be provide some examples in the file CODING_CONVENTIONS, this will give to the user an important structure about how to should to write the doxygen documentation headers
<!--
Description (as detailed as possible) of your contribution.
- describe the part/s of the code is (are) involved
- if it's a bug fix, describe the bug that it solves and how it is solved
- you can give more information about how to test your changes
-->


### Testing procedure

<!--
How should your contribution be tested? provide at least the following steps:
- which test, example or piece of code need to be compiled
- Expected output on success or error
-->


### Issues/PRs references

<!--
Use keywords with the link to the issue you want to resolve, this way some actions can perform automatically, e.g. closing an issue
- If the PR fix an issue: Close, Closes, Fix, Fixes or Resolve
- If the PR is related to another one or issue: see, see also
- If another PR need to be merged before this one: Depend on or Depends on

Examples:
  Fixes #135, see also #135, depends on #135, etc

See more about this feature: https://help.github.com/articles/closing-issues-using-keywords/.
-->

luisan00 commented on CODING_CONVENTIONS.md:
```diff
@@ -259,23 +259,23 @@ An examplary doxygen documentation in a header file can look like this.
 /*
- * Copyright (C) 2014 Peter Schmerzl <peter@schmerzl-os.org>
+ * Copyright (C) 2014 your_name yourlastname <youremailadrr@mail.com>
  *
  * This file is subject to the terms and conditions of the GNU Lesser General
  * Public License v2.1. See the file LICENSE in the top level directory for more
  * details.
  */
```
we are not using this license header

eduazocar replied:
I should use this?
*/ Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0

luisan00 replied:
Yep, I think it's better, in the same line as the proyect license,.
also may be a good Idea to put a reference for the original repo in the footer of this document, what do you think about it?

luisan00 reviewed (changes requested):
review the proposed changes please

eduazocar commented:
I've added the following updates:
update#1 with @luisan00 : we changed the authors indentifier to example
update#2:  changes with the copyrigth model
update#3: changes with the license and added a footer linked with this repo
